keyboards/preonic/rev3/rev3.c

```diff
@@ -53,11 +53,11 @@ void matrix_scan_kb(void) {
 
 #ifdef DIP_SWITCH_ENABLE
  __attribute__((weak))
-void dip_update(uint8_t index, bool active) {}
+bool dip_update(uint8_t index, bool active) { return true;}
```

Comment:
Should really get rid of this function in favour of the core API, but maybe that's a job for another PR?

Reply:
Probably should, but encoder has same issue.  Didn't want to change too much here.